django/contrib/admin/static/admin/css/base.css

```diff
@@ -20,10 +20,15 @@ a:link, a:visited {
     text-decoration: none;
 }
 
-a:hover {
+a:focus, a:hover {
     color: #036;
 }
 
+a:focus {
+    outline: none;
```

Comment:
I don't think this is good in terms of accessibility and UX. It's fine to have our own `:focus` states but I think we should respect people's expectations -- users who primarily navigate by keyboard would expect to see the same (default) tab indication  as on other sites. 

Let's discuss it.